jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/BuildService.java

```diff
@@ -194,24 +200,23 @@ private Map<String, String> addBuildArgs(JKubeConfiguration configuration) {
                 .build();
     }
 
-    private Map<String, String> addBuildArgsFromProperties(Properties properties) {
+    private static Map<String, String> addBuildArgsFromProperties(Properties properties) {
         Map<String, String> buildArgs = new HashMap<>();
         for (Object keyObj : properties.keySet()) {
             String key = (String) keyObj;
             if (key.startsWith(ARG_PREFIX)) {
                 String argKey = key.replaceFirst(ARG_PREFIX, "");
                 String value = properties.getProperty(key);
 
-                if (!isEmpty(value)) {
+                if (StringUtils.isNotBlank(value)) {
                     buildArgs.put(argKey, value);
                 }
             }
         }
-        log.debug("Build args set %s", buildArgs);
         return buildArgs;
     }
 
-    private Map<String, String> addBuildArgsFromDockerConfig() {
+    private static Map<String, String> addBuildArgsFromDockerConfig() {
```

Comment:
After revisiting this PR from https://github.com/eclipse-jkube/jkube/issues/2904 's perspective, I have a feeling that maybe it would be better if all build arg related methods could be moved to a new class `BuildArgUtil` or `BuildArgResolver` . This way we will be able to use this from OpenShiftBuildService as well.

Reply:
Yes I would leave it like that to keep it Small and Simple. And maybe have a dedicated PR for a refactor. WDYT ?

Reply:
yeah, maybe better to do it in a follow up PR